src/lib.rs

```diff
@@ -193,6 +194,12 @@ impl IronOxide {
 
     /// Create an IronOxide instance. Depends on the system having enough entropy to seed a RNG.
     fn create(curr_user: &UserResult, device_context: &DeviceContext) -> IronOxide {
+        let rt = tokio::runtime::Builder::new()
+//            .core_threads(8)  // default is num_cpus
```

Comment:
I think a comment on the `let rt` with something to the effect of `This runtime should have the core_threads set to the number of cpus on the machine, but with more blocking threads since we expect the SDK methods to block frequently on network`

Then you can remove this commented out line.

Reply:
done